packages/react-core/src/components/DescriptionList/DescriptionList.tsx

```diff
@@ -99,6 +112,12 @@ export const DescriptionList: React.FunctionComponent<DescriptionListProps> = ({
       ...setBreakpointModifiers('--pf-c-description-list--GridTemplateColumns--min', autoFitMinModifier)
     };
   }
+  if (termWidthModifier) {
+    style = {
+      ...style,
+      ...setBreakpointModifiers('--pf-c-description-list__term--width', termWidthModifier)
```

Comment:
This isn't seeming to update a style
<img width="1322" alt="Screen Shot 2022-07-20 at 12 18 18 PM" src="https://user-images.githubusercontent.com/26305082/180043512-481f4193-6032-4922-b36e-dabe2b45b1ac.png">
Shouldn't I see a reference to the --pf-c-description-list__term--width var in the styles?

Reply:
It is applied on the `<dl>` element a bit higher
![image](https://user-images.githubusercontent.com/72520867/180175831-b502095c-42dc-45b8-83a2-53009be76374.png)